test/setup/babel-register.js

```diff
@@ -0,0 +1,6 @@
+require('babel-core/register')({
+    presets: [
+        'es2015',
+        'react',
+    ],
+});
```

Comment:
The rollup-plugin-babel package has its own `es2015-babel` preset which conflicts if you also try to use the standard `es2015` Babel preset because Babel merges the collection of presets.

That said, I should probably explicitly list all of the ES2015+ syntax I want to use in the library by _not_ using presets and instead using all `plugins`.